src/test/java/bio/terra/workspace/common/fixtures/ControlledResourceFixtures.java

```diff
@@ -0,0 +1,82 @@
+package bio.terra.workspace.common.fixtures;
+
+import bio.terra.workspace.generated.model.GoogleBucketCreationParameters;
+import bio.terra.workspace.generated.model.GoogleBucketDefaultStorageClass;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycle;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycleRule;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycleRuleAction;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycleRuleActionType;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycleRuleCondition;
+import bio.terra.workspace.service.datareference.model.CloningInstructions;
+import bio.terra.workspace.service.resource.controlled.gcp.ControlledGcsBucketResource;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/** A series of static objects useful for testing controlled resources. */
+public class ControlledResourceFixtures {
+
+  public static final UUID WORKSPACE_ID = UUID.fromString("00000000-fcf0-4981-bb96-6b8dd634e7c0");
+  public static final UUID RESOURCE_ID = UUID.fromString("11111111-fcf0-4981-bb96-6b8dd634e7c0");
+  public static final UUID DATA_REFERENCE_ID =
+      UUID.fromString("33333333-fcf0-4981-bb96-6b8dd634e7c0");
+  public static final String OWNER_EMAIL = "jay@all-the-bits-thats-fit-to-blit.dev";
```

Comment:
😄

Reply:
It was going to be a .it domain, but for Italy you need paperwork saying you're an Italian business.